beacon-chain/core/state/benchmarks_test.go

```diff
@@ -16,19 +16,6 @@ import (
 
 var runAmount = 25
 
-func TestExecuteStateTransition_FullBlock(t *testing.T) {
```

Comment:
I'm removing this test. This is actually more housekeeping work than good. Whenever there's a change to a beacon object, this test requires a manual update